hooks/pre/30-upgrade.rb

```diff
@@ -141,7 +141,9 @@ def fail_and_exit(message)
                   'Please run --upgrade first, then --upgrade-puppet.'
   end
 
-  Kafo::Helpers.log_and_say :info, 'Upgrading...'
+  Kafo::Helpers.log_and_say :info, 'Upgrading, to monitor the progress on all related services, please do:'
+  Kafo::Helpers.log_and_say :info, '  foreman-tail | tee upgrade-$(date +%Y-%m-%d-%H%M).log'
+
```

Comment:
I'm not a fan of sleep. A user who expects it will be ready but those new might not expect it and miss it. On the other hand it might be a good start we can iterate on later.